tests/suites/1_stateful/08_select_stage/08_00_parquet/08_00_00_basic.sh

```diff
@@ -41,4 +41,17 @@ echo "create stage s4 FILE_FORMAT = (type = PARQUET);" | $BENDSQL_CLIENT_CONNECT
 echo "copy into @s4 from t2;" | $BENDSQL_CLIENT_CONNECT | cut -d$'\t' -f1,2
 echo "select * from @s4;" | $BENDSQL_CLIENT_CONNECT
 
+## generate large parquet files will cause timeout, we comment it now
```

Comment:
There is no test to cover this PR?

Reply:
we can use a small parquet and set parquet_fast_read_bytes to zero.

Reply:
and control rg size like this https://github.com/databendlabs/databend/blob/main/tests/data/parquet/multi_page/gen.py#L22

Reply:
Update the settings to the PR summary.